src/styles/_spacing.scss

```diff
@@ -48,7 +48,7 @@ $maxBreakpoint: map.get($grid-breakpoints, 'xl');
                     min-width: #{$spacing}px;
                 }
                 .max-w-#{$breakpoint}-#{$spacing} {
-                    max-width: #{$spacing}px;
+                    max-width: #{$spacing}px !important;
```

Comment:
Without it, this isn't getting applied to the floating box width on `UseChallengeSolutionResults.tsx` line 125 🤔 . Any thoughts for another solution?
<img width="902" alt="image" src="https://user-images.githubusercontent.com/59381432/179797891-7a5d7bf5-7b80-4174-852e-1a963cfc75d3.png">

Reply:
This has me scratching my head since the breakpoints should override. 🤔 I could have sworn I saw this working when I tested your branch and removed it but I must be seeing things.. lol. Looks good to me! 

Now I'm curious if the other classes are broken and including `!important` fixes them 🤯

Reply:
Yea I thought it should override too 🤷🏻 . I'm curious about other breaking spots too, I'll keep an 👁️ out